src/views/Practice/Practice.jsx

```diff
@@ -716,11 +716,11 @@ const Practice = () => {
           questions[currentQuestion]?.contentSourceData || [];
         const stringLengths = contentSourceData.map((item) => item.text.length);
         const length =
-          (questions[currentQuestion]?.mechanics_data &&
+          questions[currentQuestion]?.mechanics_data &&
```

Comment:
@ajinkyapandetekdi this is already merged in https://github.com/Sunbird-ALL/all-learner-ai-app/pull/191

Reply:
this code is merge in all-1.2-tn-dev not in all-1.3-tn-dev